common-custom-user-data-maven-extension/src/main/java/com/gradle/CommonCustomUserDataMavenExtension.java

```diff
@@ -29,9 +30,10 @@ public void afterProjectsRead(MavenSession session) throws MavenExecutionExcepti
         if (buildScan != null) {
             logger.debug("Capturing custom user data in build scan");
             CustomUserData.addToBuildScan(buildScan);
-            GroovyScriptUserData.addToBuildScan(session, buildScan, logger);
             logger.debug("Finished capturing custom user data in build scans");
         }
+        BuildCacheApi buildCache = BuildCacheApiAccessor.lookup(container, getClass());
```

Comment:
I think it's possible to make this backwards compatible by typing this with `Object`.

Reply:
I think it's ok to break backward compatibility here since this code also serves as a showcase for programmatic configuration.

❌ We should increment the major version number, though.

Reply:
Increased it to `1.3-SNAPSHOT`